datalad_registry/blueprints/api/dataset_urls/models.py

```diff
@@ -88,6 +102,20 @@ class QueryParams(BaseModel):
         "in the query.",
     )
 
+    return_metadata: Optional[MetadataReturnOption] = Field(
+        None,
+        description="Whether and how to return metadata of the datasets at the URLs. "
+        "If this query parameter is not provided, the `metadata` field "
+        "of each returned dataset URL object will be `null`. "
```

Comment:
would it be hard to change it that we simply do not return `metadata` field at all?  or in other words why do you prefer to return `null`/`None` instead?

Reply:
Now I see that there is an easy way to do it that I didn't realize before. I may just put in that change if I think it doesn't effect the project negatively in the long run.